composer.json

```
@@ -30,9 +30,9 @@
         "phpunit/phpunit": "^9.6",
         "psr/container": "^1.0 || ^2.0",
         "symfony/http-foundation": "^5.4 || ^6.0 || ^7.0",
-        "symfony/phpunit-bridge": "^6.3",
+        "symfony/phpunit-bridge": "^6.4 || ^7.0",
```

```suggestion
        "symfony/phpunit-bridge": "^7.0",
```

The bridge is a tool we use. No need to support multiple versions.